velox/functions/prestosql/registration/ArrayFunctionsRegistration.cpp

```diff
@@ -67,6 +67,15 @@ inline void registerArrayFrequencyFunctions() {
       Array<T>>({"array_frequency"});
 }
 
+template <typename T>
+inline void registerArrayNormalizeFunctions() {
+  registerFunction<
+      ParameterBinder<ArrayNormalizeFunction, T>,
```

Comment:
Why do we need ParameterBinder here? CC: @laithsakka

Reply:
This was done after other array functions in the file, most of them use ParameterBinder. If I try to use registerFunction directly, I get a compilation error.